server/src/test/java/io/crate/execution/dml/IndexerTest.java

```diff
@@ -335,6 +335,7 @@ public void test_default_and_generated_column_within_object() throws Exception {
     }
 
     @Test
+    @Ignore("https://github.com/crate/crate/issues/14189")
     public void test_default_for_full_object() throws Exception {
```

Comment:
Minor: Maybe we can add a comment, that this isolated test passes since it covers only part of code path but actually running `create table tbl (x int, o object as (x int) default {x=10});` shows an error